pkg/sql/catalog/systemschema/system.go

```diff
@@ -1007,6 +1007,149 @@ CREATE TABLE system.span_stats_tenant_boundaries (
 	  FAMILY "primary" (crdb_region, unavailable_at)
 ) ;
 `
+
+	TxnExecutionStatsTableSchema = `
+	CREATE TABLE system.txn_exec_insights (
```

Comment:
rename table to `transaction_execution_insights` wo abbreviation.

Reply:
`query` field was removed here. Add it back to avoid quering this info from statement insights table.